src/fenix.py

```diff
@@ -11,14 +11,15 @@
 #
 def update_android_components(ac_repo, fenix_repo, author, debug, dry_run):
     update_android_components_nightly(
-        ac_repo, fenix_repo, author, debug, "main", dry_run
+        ac_repo, fenix_repo, "", author, debug, "main", dry_run
```

Comment:
Nit: let's use keyword arguments to make clearer what this empty string represents. Let's change all calls this way, I think.

```suggestion
        ac_repo,
        fenix_repo,
        target_path="",
        author=author,
        debug=debug,
        release_branch_name="main",
        dry_run=dry_run,
```

(Sadly Python doesn't allow to use positional arguments once the first keyword arg is given)